indexstylesheet.css

```diff
@@ -0,0 +1,411 @@
+@import url('https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700;800;900&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'M PLUS Rounded 1c', sans-serif;
+    background-color: white;
+    padding: none;
+    margin: none;
+}
+
+img {
+    border-radius: 15px;
+}
+
+/* ****************** LINKS ********************** */
+a {    
+    margin: 0;
+    padding: 10px;
+    width: 200px;
+    border-radius: 5px;
+    background-color: rgb(153, 247, 59);
+    font-family: 'M PLUS Rounded 1c', sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+}
+
+a:link {
+    color: black;
+}
+
+a:visited {
+    color: lightslategray
+}
+
+a:hover {
+    color: rgb(10, 189, 10)
+}
+
+a:active {
+    color: yellow
+}
+
+a.mainlink {
+    text-decoration: none;
+    background-color: white;
+}
+
+/* *************grid layout ************ */
+.indexcontainer {
+    display: grid;
+    grid-gap: 1rem;
+    grid-template-columns: 200px 200px 200px 200px 200px 200px 200px 200px;
+    grid-template-rows: 170px 35px 400px 35px 350px;
+}
+
+/* .header {
+    grid-column: 3/7;
+    grid-row: 1/2;
+    display: flex;
+    flex-direction: row;
+    justify-self: center;
+} */
+
+.line {
+    grid-column: 3/7;
+    grid-row: 2/3;
+}
+
+/* .features {
+    grid-column: 3/7;
+    grid-row: 3/4;
+    display: grid;
+    justify-content: center;
+} */
+
+.line2 {
+    grid-row: 4/5;
+}
+
+/* .hero {
+    grid-column: 1/9;
+    grid-row: 5/6;
+    display: grid;
+    grid-rows: 2fr 1fr;
+    height: 400px;
+} */
+
+/* .main {
+    grid-column: 3/7;
+    grid-row: 5/7;
+    display: grid;
+    grid-template-columns: 100px 400px 200px 100px;
+    grid-template-rows: auto auto auto;
+    grid-template-areas: 
+    "quote quote quote quote"
+    ". blog2 peek4 ."
+    ". blog3 peek5 .";
+    grid-gap: 8px;
+} */
+
+/* .footer {
+    background-color: rgb(207, 212, 216);
+    grid-column: 1/9;
+    grid-row: 7/8;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 1fr 3fr;
+    grid-template-areas:
+    ". footer1 footer1 ."
+    ". form buttons ."
+} */
+
+
+
+/* *************** HEADER ***************** */
+
+.header {
+    grid-column: 3/7;
+    grid-row: 1/2;
```

Comment:
If you can, try pulling up your projects on different sized monitors, because that'll allow you to see the responsiveness. With that being said, I think this sizing is messing with your header on the main page because its not centered